com.gamefuse.sdk/Runtime/FriendshipService/FriendshipService.cs

```diff
@@ -0,0 +1,87 @@
+using UnityEngine;
+using UnityEngine.Networking;
+using System.Threading.Tasks;
+
+namespace GameFuseCSharp
+{
+    public class FriendshipService : AbstractService, IFriendshipService
+    {
+        public FriendshipService(string baseUrl, string token)
+        {
+            _baseUrl = baseUrl;
+            _token = token;
+        }
+
+        public async Task<FriendRequestResponse> SendFriendRequestAsync(string username)
+        {
+            string url = $"{_baseUrl}/friendships";
+            string jsonBody = JsonUtility.ToJson(new FriendRequestData { username = username });
+
+            using (UnityWebRequest webRequest = CreateRequest(url,HttpVerbs.POST,jsonBody))
+            {
+                return await SendRequestAsync<FriendRequestResponse>(webRequest);
+            }
+        }
+
+        public async Task<FriendshipStatusResponse> UpdateFriendRequestStatusAsync(int friendshipId, string status)
```

Comment:
This isnt super clear what this is used for.  We should make sure user exposed methods that will be in the docs should be pretty obvious what they are for.  Maybe make it private and then expose public methods to accept it or whatever

Reply:
Yeah I like this better too, I think I remember being on the fence about this b/c I had put Decline and Accept methods on the GameFuseUser class but if a dev decides to use the friendship service on on it's own they wouldn't see those methods.. well now they will :)